firmware/common/portapack_persistent_memory.cpp

```diff
@@ -820,6 +820,16 @@ void set_encoder_dial_sensitivity(uint8_t v) {
     data->encoder_dial_sensitivity = v;
 }
 
+bool save_load_pmem_from_sdcard_flag() {
+    File pmem_flag_file_handle;
+    std::string pmem_flag_file = "/SETTINGS/PMEM_FILEFLAG";
+    auto result = pmem_flag_file_handle.open(pmem_flag_file);
+    if (!result.is_valid()) {
```

Comment:
is_valid is already a bool. Maybe `return result.is_valid();`

Reply:
Actually, what is this even doing? It's just returning true if the file opens?

Reply:
If you just want to verify the file exists, there's a `file_exists` helper I added to file.hpp.

Reply:
I think the word "save" in the name is confusing me.
Maybe just `should_load_pmem_from_sdcard` or even `has_sdcard_pmem_file` would describe what this is doing better.

Reply:
I reduced the function to return file_exists( PMEM_FLAGFILE )
Changed the name of the test func to should_use_sdcard_for_pmem